### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by LeoSpartaDeV**
https://leospartadev.github.io/layout_html-form/
https://leospartadev.github.io/layout_html-form/report/html_report/

### CODE REVIEW

**Comment by LeoSpartaDeV** [comment]
https://leospartadev.github.io/layout_html-form/
https://leospartadev.github.io/layout_html-form/report/html_report/

File: src/index.html [hunk 1]
```diff
@@ -1,17 +1,210 @@
...
+        </div>
+      </fieldset>
+
+      <fieldset class="contacts__form-field">
+
+        <legend>An interesting fact about you!</legend>
+
+        <div class="contacts__form-item">Do you love cats?
+
+          <label>
```
Comment (ivanchuk-yurii) [inline src/index.html:115]:
```suggestion
      <fieldset class="contacts__form-field">
        <legend>An interesting fact about you!</legend>
        <div class="contacts__form-item">Do you love cats?
          <label>
```

File: src/index.html [hunk 2]
```diff
@@ -1,17 +1,210 @@
...
+          </label>
+        </div>
+        <div class="contacts__form-item">
+          <label>What is your favorit color?
+            <input
+              type="color"
+              name="color"
+            >
+
+          </label>
```
Comment (ivanchuk-yurii) [inline src/index.html:140]:
```suggestion
            >
          </label>
```

File: src/index.html [hunk 3]
```diff
@@ -1,17 +1,210 @@
...
+        <div class="contacts__form-item">
+          <label for="textarea">
+            Comments:
+          </label>
+          <textarea
+            name="textarea"
+            rows="2"
+            id="textarea"
+          >
+          </textarea>
```
Comment (ivanchuk-yurii) [inline src/index.html:186]:
```suggestion
          ></textarea>
```
image

Comment (ivanchuk-yurii) [reply src/index.html:186]: to fix this

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 3 above)
Comment (LeoSpartaDeV) [reply src/index.html:186]: > 

error  Tag’s closing bracket should be on  the same level as opening one   (closing-and-opening-on-seme-level)

Comment (LeoSpartaDeV) [reply src/index.html:186]: html lint error

**Comment by LeoSpartaDeV** [comment]
+

File: src/index.html [hunk 4]
```diff
@@ -1,17 +1,206 @@
...
+          What are your favorite brand of cars?
+          <select
+            name="brans"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="AUDI">Audi</option>
+            <option value="LADA">Lada</option>
+          </select>
+        </div>
```
Comment (DarkMistyRoom) [inline src/index.html:158]: Why don't you use a label here?

File: src/index.html [hunk 5]
```diff
@@ -1,17 +1,206 @@
...
+
+        <div class="contacts__form-item">
+          How do you rate our work?
+          <input
+            type="range"
+            name="range"
+            value="0"
+          >
+        </div>
```
Comment (DarkMistyRoom) [inline src/index.html:167]: And here?

File: src/index.html (hunk 3 above)
Comment (DarkMistyRoom) [reply src/index.html]: Yes, but it is just a linter cofig. If it is important for the element's behavior, you can skip it

**Pull request closed without merging**